packages/lib-classifier/src/hooks/useTranscriptionReductions.js

```diff
@@ -23,8 +23,11 @@ export default function useTranscriptionReductions() {
     }
   } = useStores()
 
-  const { loaded, caesarReductions } = useCaesarReductions(workflow.caesarReducer)
+  // NOTE: useFreehandLineReductions() and useTranscriptionReductions() are called regardless of reducer key
+  if (workflow.caesarReducer !== 'alice') return
```

Comment:
Same comment here about breaking the return value of these hooks. This change might have broken transcription projects.

Reply:
This breaks [the rules of hooks](https://react.dev/warnings/invalid-hook-call-warning) too. React should error about this in the console.